packages/get-platform/src/getPlatform.ts

```diff
@@ -289,10 +390,11 @@ ${additionalMessage}`,
 
   // sometimes we fail to detect the distro in use, so we default to debian
   const defaultDistro = 'debian' as const
-  if (platform === 'linux' && distro === undefined) {
+  if (platform === 'linux' && targetDistro === undefined) {
     warnOnce(
       'distro:undefined',
-      `Prisma failed to detect the Linux distro in use, and may not work as expected. Defaulting to "${defaultDistro}".`,
+      `Prisma doesn't know which engines to download for the Linux distro "${originalDistro}". Falling back to Prisma engines built for "${defaultDistro}". While these engines are likely to work on your system, Prisma may not work as expected.
+Please run \`prisma generate\` and \`prisma db push\` to chat that Prisma is working correctly on your system.`,
     )
   }
```

Comment:
Both `originalDistro` and `targetDistro` are used in this warning message, which also now clarifies that Prisma will likely still work. The warning is now "softer".

Reply:
How about we make it a _lot_ shorter and jsut add a pris.ly link to some documentation that explains what this means, and how to test if things work?

This will be very common output for some users (all on not explicitly supported Linux distros). We should make it shorter, not even longer and more complex.

Reply:
I'm reverting the change to the warning's wording. We can change that in a separate PR, so that we can at least merge the code for the rest of this PR